source/class/cv/TemplateEngine.js

```diff
@@ -347,6 +348,40 @@ qx.Class.define('cv.TemplateEngine', {
       await cv.Application.structureController.preParse(xml);
     },
 
+    generateManifest() {
+      const color = getComputedStyle(document.body).getPropertyValue('background-color');
+      let baseUrl = window.location.pathname;
+      if (baseUrl.endsWith('/index.html')) {
+        const parts = baseUrl.split('/');
+        parts.pop();
+        baseUrl = parts.join('/') + '/';
+      }
+      baseUrl = window.location.origin + baseUrl;
+
+      const startUrl = window.location.origin + window.location.pathname + window.location.search;
```

Comment:
I think we should add `window.location.hash` as well, so that you can start the installed visu on a sub page, when desired

Reply:
I left that one out on purpose. I think most of the users will want to start with the first page. In addition to that I think that it will lead to lots of "support cases" where users that hit the install button while being on a subpage, will be confused because the "app" will not start on the start page.